src/heliaServer.ts

```diff
@@ -26,30 +31,47 @@ interface EntryParams {
   '*': string
 }
 
+interface HeliaPathParts {
+  address: string
+  namespace: string
+  relativePath: string
+}
+
+interface HeliaFetchOptions extends HeliaPathParts {
+  signal?: AbortSignal
+  cid?: CID | null
+  peerId?: PeerId | null
+}
+
 export class HeliaServer {
-  private heliaFetch!: HeliaFetch
+  private heliaFetch!: Awaited<ReturnType<typeof createVerifiedFetch>>
```

Comment:
Why the complicated type?  Just importing [VerifiedFetch](https://ipfs.github.io/helia-verified-fetch/interfaces/VerifiedFetch.html) from `@helia/verified-fetch` will do?

Reply:
this PR is significantly outdated :(

Reply:
resolved with latest deps update